samples/client/petstore/elixir/mix.exs

```diff
@@ -36,6 +36,7 @@ defmodule OpenapiPetstore.Mixfile do
   defp deps do
     [
       {:tesla, "~> 1.7"},
+      {:ecto, "~> 3.12"},
```

Comment:
I considered using [TypedEctoSchema](https://hexdocs.pm/typed_ecto_schema/TypedEctoSchema.html), which would have resulted in more compact models - but I figured it's best to stick to the most common and standard libraries. Since models are automatically generated, I feel `TypedEctoSchema` main value proposition does not apply.